http-endpoint-example/README.md

```diff
@@ -0,0 +1,46 @@
+# How to download Met Office atmospheric model data using an HTTP endpoint
+A demonstration code example of downloading objects in response to SNS messages via an HTTP endpoint.
+
+By using the following you will be able to subscribe to an Amazon Web Services (AWS) Simple Notification Service (SNS) topic and start downloading objects.
+The running python script will accept notifications from the SNS topic and download the objects according to the input parameters.
+
+
+## Prerequisites:
+
+  * A service API key
+  * A Python 3 environment containing the flask and requests library.
+  * A server with a Public IP or DNS address with an open port. Ports other than standard http/https are not reconmmended as described [here](https://aws.amazon.com/sns/faqs/ "AWS documentation")
+  * If used in production a method of verifying the messages are from AWS, this can be found [here](http://docs.aws.amazon.com/sns/latest/dg/SendMessageToHttp.verify.signature.html "AWS documentation")
+
+# Actions:
+1. ### Clone repo onto the server
+
+2. ### Start listening for notifications
+   To start polling the SNS topic by running the host.py with the following command
```

Comment:
Grammar police again.... "Start polling" i.e. delete the "To" at the beginning, and I think it should be "running host.py with the following.." i.e. delete "the". :police_car: :smiling_imp: